server.ts

```diff
@@ -33,6 +33,14 @@ app.post('/api/graphql', async (req, res) => {
     return;
 });
 
+app.post('/api/getSignaturesForAddress', async(req, res) => {
+    const { address } = req.body;
+
+    let results = await rpc.getSignaturesForAddress(address).send();
+    const signatures = results.map(result => {return { signature: result.signature }});
+    res.send(JSON.stringify(signatures));
+});
+
```

Comment:
We should probably use a `try-catch` here to return any errors to the app with a proper server code. I don't use `try-catch` above in the GraphQL response, since GraphQL errors don't throw but are instead returned in the `errors` field.